gc/index.html

```diff
@@ -327,7 +327,7 @@
             <em> Partner earned </em>
             <input class="form-control" type="text" name="f-p-earned" id="f-p-earned" />
             </label>
-            <label for="f-delivery-fees">
+            <label for="f-c-fees">
```

Comment:
pourquoi `f-c-fees` ?
tu n'as pas répercuté le nouveau nom dans l'`input` en dessous.
le `for="f-c-fees"` permet juste, quand tu es sur la page, que lorsque tu cliques sur le label, ça te met le curseur sur l'input associé
Sauf qu'il faut pour cela que l'attribut `name`de l'input soit le même.